Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -263,6 +263,34 @@ void OnChildrenCollectionChanged(object sender, NotifyCollectionChangedEventArgs
 			UpdateIgnoreContainerAreas();
 		}
 
+		void TeardownPage(Page page, int i)
+		{
+			page.PropertyChanged -= OnPagePropertyChanged;
+		}
+
+		void SetupPage(Page page, int i)
+		{
+			page.PropertyChanged += OnPagePropertyChanged;
+		}
+
+		void Reset()
+		{
+			var i = 0;
+			foreach (var page in Element.Children)
+				SetupPage(page, i++);
+		}
+
+		private void OnPagePropertyChanged(object sender, PropertyChangedEventArgs e)
+		{
+			if (e.PropertyName == Page.TitleProperty.PropertyName)
+			{
+				var page = (Page)sender;
+				var index = Element.Children.IndexOf(page);
```

Comment:
On dispose you need to remove the events of the pages @jimmgarrido , can you try with that.